DeepLearningOnShallowHardware.md

```diff
@@ -1,10 +1,20 @@
 # Deep Learning on Shallow Hardware
 
+## Project Name 
 
-| Project name                  | Deep Learning on Shallow Hardware       
-| Project leader                | Adnan Siddiqui
-| Project leader Slack username | @siddiqaa    
-| Project slack channel         | NA (we just use email)           
+Deep Learning on Shallow Hardware       
+
+## Project Leader
+
+Adnan Siddiqui
+
+## Project Leader Slack Username
+
+@siddiqaa    
```

Comment:
Isn't your Slack username `@adnans`?

Reply:
no @siddiqaa

Reply:
oh wait - yes - adnans - got my github and slack confused